README.md

```diff
@@ -103,12 +111,32 @@ Remove-PowerBIWorkspaceUser -Scope Organization -Id 3244f1c1-01cf-457f-9383-6035
 
 ### Restore Workspace
 
+To view deleted workspaces as a tenant administrator:
+
+```powershell
+Get-PowerBIWorkspace -Scope Organization -Deleted -All
+```
+
 Restores a deleted workspace:
 
 ```powershell
 Restore-PowerBIWorkspace -Id "3244f1c1-01cf-457f-9383-6035e4950fdc" -RestoredName "TestWorkspace" -AdminEmailAddress "john@contoso.com"
 ```
 
+### Repair orphaned workspaces
+
+Workspaces can become orphaned when no administrators are assigned to the workspace, to view you must be a tenant administrator:
```

Comment:
"A workspace becomes orphaned when it has no assigned administrators. If you are a tenant administrator, run the following to view all orphaned workspaces:"